<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findsales</class>
 <widget class="QDialog" name="findsales">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>458</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>717</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>717</width>
    <height>458</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询经办人信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
     </property>
     <property name="text">
      <string>查询经办人信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6,1">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>请选择查询方式</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按编号查询</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按姓名查询</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按性别查询</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按电话查询</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按备注查询</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
